code/modules/telescience/radiostation.dm

```diff
@@ -236,7 +239,7 @@
 // Record player
 /obj/submachine/record_player
 	name = "record player"
-	desc = "An old school vinyl record player sat on a set of drawers. Shame you don't have any records."
+	desc = "An old school vinyl record player sat on a set of drawers. Shame you don't have any records. There's a small \"out of order\" label on the casing."
```

Comment:
why not dynamically append the description if the define is true instead of having to make two changes?